src/github.com/matrix-org/dendrite/clientapi/writers/createroom.go

```diff
@@ -0,0 +1,42 @@
+package writers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/matrix-org/dendrite/clientapi/auth"
+	"github.com/matrix-org/dendrite/clientapi/common"
+	"github.com/matrix-org/util"
+)
+
+// http://matrix.org/docs/spec/client_server/r0.2.0.html#post-matrix-client-r0-createroom
```

Comment:
Minor nit. Can we link to the https:// docs? The green padlock gives me warm fuzzy feelings.